packages/runtime-core/src/scheduler.ts

```diff
@@ -11,34 +11,33 @@ export function nextTick(fn?: () => void): Promise<void> {
 }
 
 export function queueJob(job: () => void) {
-  if (queue.indexOf(job) === -1) {
+  if (!queue.includes(job)) {
     queue.push(job)
     if (!isFlushing) {
       nextTick(flushJobs)
     }
   }
 }
 
-export function queuePostFlushCb(cb: Function | Function[]) {
-  if (Array.isArray(cb)) {
-    postFlushCbs.push.apply(postFlushCbs, cb)
```

Comment:
I'd rather just do `postFlushCbs.push(...cb)` here to avoid allocating an extra array.